test/Microsoft.AspNetCore.Blazor.Build.Test/TestFiles/DesignTimeCodeGenerationTest/ChildComponent_GenericBind/TestComponent.codegen.cs

```diff
@@ -33,14 +33,15 @@ protected override void BuildRenderTree(Microsoft.AspNetCore.Blazor.RenderTree.R
 #line default
 #line hidden
             );
-            __o = Microsoft.AspNetCore.Blazor.Components.BindMethods.GetValue(
+            __o = Microsoft.AspNetCore.Blazor.Components.RuntimeHelpers.TypeCheck<string>(Microsoft.AspNetCore.Blazor.Components.BindMethods.GetValue(
 #line 2 "x:\dir\subdir\Test\TestComponent.cshtml"
                                     Value
 
 #line default
 #line hidden
-            );
-            __o = Microsoft.AspNetCore.Blazor.Components.BindMethods.SetValueHandler(__value => Value = __value, Value);
+            ));
+            __o = new System.Action<string>(
```

Comment:
This doesn't seem right.

Reply:
Ah yes. This was a test bug. This test was testing the case where you were using bind on a parameter that isn't defined. I fixed this test, added another to cover that scenario. So these diffs are expected.